plinth/modules/users/urls.py

```diff
@@ -39,9 +43,8 @@
         non_admin_view(views.UserChangePassword.as_view()),
         name='change_password'),
     # Add Django's login/logout urls
-    url(r'^accounts/login/$', public(auth_views.login),
-        {'template_name': 'login.html'}, name='login'),
-    url(r'^accounts/logout/$', public(auth_views.logout),
+    url(r'^accounts/login/$', public(sso_login), name='login'),
+    url(r'^accounts/logout/$', public(sso_logout),
```

Comment:
The SSO login simply calls Django's authentication method and adds a cookie to the response. So, we're still using Django auth.
SSO is currently an essential module. Are you suggesting we make it optional and make it installable like an app?